apps/web-giddh/src/app/shared/template-froala/template-froala.component.ts

```
@@ -377,6 +615,74 @@ export class TemplateFroalaComponent implements OnInit {
+        this.setToCcBcc(this.customTriggerForm);
+        this.customTriggerForm.markAllAsTouched();
+
+        if (this.customTriggerForm.invalid) {
+            this.showFormValidityError();
+            return;
+        }
+
+        const formValue = cloneDeep(this.customTriggerForm.value);
+
+        if (!formValue.executionTime.dayOfWeek) {
+            delete formValue.executionTime.dayOfWeek;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)